user/serializers.py

```diff
@@ -22,3 +23,28 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.EmailField(
+        style={"input_type": "email"},
+        label=_("Email")
+    )
+    password = serializers.CharField(
+        style={"input_type": "password"},
+        write_only=True,
+        label=_("Password")
+    )
+
+    def validate(self, attrs):
+        email = attrs.get("email")
+        password = attrs.get("password")
+        user = authenticate(request=self.context.get("request"),
+                            email=email,
+                            password=password)
```

Comment:
it is better to check email/password existence before authentication and raise validation error if not